### PR TITLE
Move gPlayerFacingPosition out of sym_ewram.txt

### PR DESCRIPTION
**Pull request by Phlosioneer**

### CODE REVIEW

File: sym_ewram.txt [hunk 1]
```diff
@@ -174,9 +174,7 @@ gUnknown_02022D00: @ 2022D00
 	.include "src/slot_machine.o"
 	.include "src/battle_ai_script_commands.o"
 	
-.align 2
-gPlayerFacingPosition: @ 203AB40
-	.space 0x8
+    .include "src/fldeff_misc.o"
```
Comment (garakmon) [inline sym_ewram.txt]: tab :P

File: src/fldeff_cut.c [hunk 2]
```diff
@@ -21,6 +21,8 @@
 #include "constants/field_effects.h"
 #include "constants/songs.h"
 
+extern bool8 IsMewPlayingHideAndSeek(void);
```
Comment (huderlem) [inline src/fldeff_cut.c]: super small thing, but this function comes from `include/faraway_island.h`

Comment (Phlosioneer) [reply src/fldeff_cut.c]: Ah. It was sorta put there during the merge process.

**Comment by Phlosioneer** [comment]
This PR is updated

**Pull request merged**